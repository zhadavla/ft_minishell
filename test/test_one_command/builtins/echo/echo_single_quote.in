echo 'hello {$USER} from {$PWD}'