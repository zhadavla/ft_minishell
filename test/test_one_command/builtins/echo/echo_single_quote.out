minishell$ echo 'hello {$USER} from {$PWD}'
hello {$USER} from {$PWD}
minishell$ exit
